sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -125,15 +120,13 @@ def __init__(
         # Although we might normally thread through **kwargs this causes
         # problems as MessageProperties won't absorb spurious args.
         self._encoding = kwargs.pop("encoding", "UTF-8")
+        self._uamqp_message = None
```

Comment:
In EH, since we moved all of the uamqp/pyamqp Message creation out to right before sending (so that the user doesn't have to specify the transport to the EventData object each time they create one), how will we know whether the message property should return a uamqp or pyamqp Message?

Should we always have the message property return a pyamqp Message for now, since it will essentially provide the same info as the uamqp Message?